presto-docs/src/main/sphinx/release/release-328.rst

```diff
@@ -0,0 +1,63 @@
+========================
+Release 328 (6 Jan 2020)
+========================
+
+General Changes
+---------------
+
+* Fix correctness issue for certain correlated JOIN queries when the subquery on the
+  right produces no rows. (:issue:`1969`)
+* Fix incorrect handling of multi-byte characters when pattern is empty. (:issue:`2313`)
+* Fix failure when join criteria contains columns of different types. (:issue:`2320`)
+* Fix failure for complex outer join queries when dynamic filtering is enabled. (:issue:`2363`)
+* Improve support for correlated queries. (:issue:`1969`)
+* Support narrowing type coercions for insert queries. (:issue:`2061`)
+* Add ``regexp_count`` and ``regexp_position`` SQL functions. (:issue:`2136`)
+
+Security Changes
+----------------
+
+* Fix a security issue allowing users to contain unauthorized access to Presto cluster
+  when using password authenticator with LDAP. (:issue:`2356`)
+* Add support for LDAP referrals in LDAP password authenticator. (:issue:`2354`)
+
+JDBC Driver
+-----------
+
+* Fix behavior of ``java.sql.Connection#commit()`` and ``java.sql.Connection#rollback()``
+  methods when no statements performed in a transaction. Previously these methods
+  would fail. (:issue:`2339`)
+* Fix failure when restoring autocommit mode with
+  ``java.sql.Connection#setAutocommit()`` (:issue:`2338`)
+
+Hive Connector Changes
+----------------------
+
+* Reduce query latency and Hive metastore load when ``optimizer.join-reordering-strategy``
+  is set to ``AUTOMATIC``. (:issue:`2184`)
+* Allow configuring ``hive.max-outstanding-splits-size`` to values bigger than
+  2 GB (2,147,483,647 bytes). (:issue:`2395`)
```

Comment:
No need to write out the byte count (just "2GB" is fine). Remove space between "2" and "GB" since that matches the way we normally write (and configure) such values.